mlflow/openai/__init__.py

```diff
@@ -844,3 +862,31 @@ def load_model(model_uri, dst_path=None):
     _add_code_from_conf_to_system_path(local_model_path, flavor_conf)
     model_data_path = os.path.join(local_model_path, flavor_conf.get("data", MODEL_FILENAME))
     return _load_model(model_data_path)
+
+
+@experimental
+@autologging_integration(FLAVOR_NAME)
+def autolog(
+    log_input_examples=False,
+    log_model_signatures=False,
+    log_models=False,
+    log_datasets=False,
+    disable=False,
+    exclusive=False,
+    disable_for_unsupported_versions=True,
+    silent=False,
+    registered_model_name=None,
+    extra_tags=None,
+):
+    with contextlib.suppress(ImportError):
```

Comment:
Is it possible to throw if the openai version is < 1.0?

Reply:
Sure. We will not support openai < 1.0 then?

Reply:
I don't think we need to. < 1.0 is completely different from >= 1.0. I'd consider supporting < 1.0 if there is a request.

Reply:
Makes sense. I will do it